src/main/java/pl/piekoszek/gorskimatches/facebook/FacebookController.java

```diff
@@ -0,0 +1,30 @@
+package pl.piekoszek.gorskimatches.facebook;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("web")
```

Comment:
why web? we always start our endpoints with api, webhook would also be acceptable, anyway it should contain second part of path, like /api/webhook/facebook/page/message or sth like that